openapi/openapi.yaml

```diff
@@ -111,6 +111,30 @@ paths:
           $ref: "#/components/responses/NotFound"
         '500':
           $ref: "#/components/responses/InternalServerError"
+  /city-council-districts:
```

Comment:
This should not be part of the implementation PR. But once you pull changes from https://github.com/NYCPlanning/ae-zoning-api/pull/295, you should remove the 🚧 as part of this PR

Reply:
yeah, this was only bc I was blocked without having the documentation so I branched off from #295 to create this one.